var/spack/repos/builtin/packages/fenics/package.py

```diff
@@ -10,26 +10,44 @@ class Fenics(CMakePackage):
     """FEniCS is organized as a collection of interoperable components
     that together form the FEniCS Project. These components include
     the problem-solving environment DOLFIN, the form compiler FFC, the
-    finite element tabulator FIAT, the just-in-time compiler Instant,
-    the code generation interface UFC, the form language UFL and a
-    range of additional components."""
+    finite element tabulator FIAT, the just-in-time compiler Instant / Dijitso,
+    the code generation interface UFC, the form language UFL and a range of
+    additional components."""
 
     homepage = "http://fenicsproject.org/"
+    git      = "https://bitbucket.org/fenics-project/dolfin.git"
     url      = "https://bitbucket.org/fenics-project/dolfin/downloads/dolfin-1.6.0.tar.gz"
     base_url = "https://bitbucket.org/fenics-project/{pkg}/downloads/{pkg}-{version}.tar.gz"
```

Comment:
It seems like this variable is no longer used?

Reply:
yes, `base_url` can be removed